GOVERNACE.md

```diff
@@ -1,45 +1,92 @@
+The project is comprised of the following types of subgroups:
+
+- Project Management Committee, PMC
+- Organizer Committee
+- Developer Group
+    - Special Interest Groups, SIGs
+    - Working Groups, WGs
+
+### Project Management Committee
+
+The PMC functions as the core management team that oversees the TiKV community. It consists of those individuals identified as Maintainers on the development site. The PMC has additional responsibilities over and above those of a committer. These responsibilities ensure the smooth running of the project. PMC members are expected to review code contributions, participate in strategic planning, approve changes to the governance model and manage the copyrights within the project outputs.
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new committers, and makes all major decisions for the future with respect to TiKV. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives. This list is used for sensitive issues, such as votes for new committers and legal matters that cannot be discussed in public. It is never used for project management or planning.
 
-Developer Group:
+Membership of the PMC is by invitation from the existing PMC members. A nomination will result in discussion and then a vote by the existing PMC members. PMC membership votes are subject to consensus approval of the current PMC members.
 
-- Maintainer: Mainatiners are the planners and designers of the project, with the authority to merge branches into the master. 
-- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+The current list of PMC members are:
 
-Organizer Committee:  The Organizer Committee consists of organizers in charge of event or activity operations; in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions
+- Max Liu(**[@goroutine](https://github.com/ngaut/)**)
```

Comment:
I will propose the PMC List in this PR, https://github.com/tikv/community/pull/12 , and I create a new page to show the list. So I think there is no need to show the list here, otherwise, we need to maintain it in two places.

Reply:
OK~ I will add the link here instead.